grammars/grammar-tact.json

```diff
@@ -191,7 +191,7 @@
       "name": "keyword.operator.binary"
     },
     "operator-assignment": {
-      "match": "(\\:?=)",
+      "match": "(\\:?=[^\\!])",
```

Comment:
Wait, this looks overly specific to the test case I mentioned

Reply:
hmm, according to operator-logic it will be colored all `=` showed in {:tact}, even I disable the match of `operator-logic`, we still can see `=` is colored here, seems the match of operator-assignment is also have some issue. 

<img width="319" alt="2024-04-10 16 24 07" src="https://github.com/tact-lang/tact-docs/assets/23052633/c1f4ffb6-4259-4976-ace1-aa23a948fcef">

so, if we don't want any `=` in `=!{:tact}` or similar thing like that is colored, this kind of restrict is necessary, in this situation can we consider only when there is one space showed behind the `:` or `:=` will be match and colored?  if so we can change this to match to `(\\:?=\\s)`